### DISCUSSION TITLE
3.2.0

### DISCUSSION THREAD

**Discussion by akallabeth** [FreeRDP/FreeRDP#9783] [Q&A]
This release mostly addresses issues reported since the last release.
 Fixing some usablity and build issues as well as adding API functions
 that are needed from external projects
 
 Noteworthy changes:
 * Fix proxy module load check (#9777)
 * Improve kerberos error logging (#9771)
 * Improve mac client keyboard handling (#9767)
 * Add option to run client dynamic channel synchronous (#9764)
 * Move huge struct to heap (#9763)
 * Improved failure logging of license module (#9759)
 * Improve server side gfx logging (#9757)
 * Print shadow server help with printf instead of WLog (#9756)
 * Fix SDL client timer initialization (#9754)
 * Fix server peer message parsing (#9751)
 * Enable NEON instructions if __ARM_NEON is defined (#9748)
 * Add new proxy config file option TlsSecLevel (#9741)
 * Improve android and mac os build scripts (#9735)
 * Do not disable wayland support on BSD (#9730)
 * Fix issues with assistance file parsing (#9727, #9728)
 * Keyboard handling fixes for wayland client (#9725)
 * Fix relative pkg-config file paths (#9720)
 * Add new transport IO callback GetPublicKey (#9719)
 * Fix wayland client scaling (#9715)
 
 For a complete and detailed change log since the last release run:
 git log 3.2.0...3.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeRDP/FreeRDP/releases/tag/3.2.0'>3.2.0</a>.</em>

**Comment by ghost** [comment]
Hello there, can you please release the zip with binaries? The hashes are there, but not the zip itself.

**Reply by akallabeth** [reply]
oh, looks like I´ve missed that file.
you can always get them from our primary source http://pub.freerdp.com/releases/ 
anyway, fixed.
